client/payment-details/timeline/map-events.js

```diff
@@ -194,9 +195,9 @@ const composeFeeString = ( event ) => {
 
 	return sprintf(
 		/* translators: %1$s is the total fee amount, %2$f%% is the fee percentage, and %3$s is the fixed fee amount. */
```

Comment:
Interesting that this comment had been inconsistent with the code. Notice `%2$f%%` and `%2$.1f%%` (before your change). Your change aligns with the comment =)